multimodal/setup.py

```diff
@@ -83,11 +83,6 @@
         "configs/environment/*.yaml",
         "configs/distiller/*.yaml",
         "configs/matcher/*.yaml",
```

Comment:
If we put `__init__.py` in `configs`, can we remove the yaml files here?

Reply:
yes